README.md

```diff
@@ -149,7 +149,7 @@ And if you want to show your gravatar, just add the `gravatar` boolean property:
 }
 ```
 
-Note that the gravatar requires the email property. You also need to check the compatibility of the chosen theme. Currently, only the default theme supports Gravatar.
+Note that the gravatar requires the email property. You also need to check the compatibility of the chosen theme. Currently, only the default theme, material theme, 8bits theme and so forth supports Gravatar.
```

Comment:
Adding "so forth" doesn't make the statement any much more informative. Just remove the sentence

```suggestion
Note that the gravatar option requires the email property. You also need to check the compatibility of the chosen theme.
```

Reply:
Thank you!